tests/e2e/targetallocator-features/00-install.yaml

```diff
@@ -53,10 +53,16 @@ spec:
       prometheus:
         config:
           scrape_configs:
-            - job_name: 'otel-collector'
-              scrape_interval: 10s
-              static_configs:
-                - targets: [ '0.0.0.0:8888' ]
+          - job_name: 'otel-collector'
+            scrape_interval: 10s
+            static_configs:
+            - targets: [ '0.0.0.0:8888' ]
+            relabel_configs:
+            - regex: __meta_kubernetes_node_label_(.+)
```

Comment:
for completeness, can we also add in a labelkeep action here too?

Reply:
@jaronoff97 Done!